scripts/photon-containers.sh

```diff
@@ -19,8 +12,7 @@ k8s.gcr.io/kube-proxy:v1.14.9
 k8s.gcr.io/pause:3.1
 k8s.gcr.io/etcd:3.3.10
 k8s.gcr.io/coredns:1.3.1
-docker.io/weaveworks/weave-kube:2.6.2
-docker.io/weaveworks/weave-npc:2.6.2
+antrea/antrea-ubuntu:v0.6.0
```

Comment:
Assuming you also tested without network connectivity? If not, did you see the local image being used, i.e. no pull?

Reply:
Yup and Antrea YAML includes `IfNotPresent` policy